chapters/ru/events/1.mdx

```diff
@@ -0,0 +1,48 @@
+# Занятия и семинары[[live-sessions-and-workshops]]
+
+В связи с выходом первой и второй частей курса мы организовали несколько занятий и семинаров по программированию. Ссылки на записи этих сессий и семинаров вы можете найти ниже.
+
+## Занятия по программированию[[live-coding-sessions]]
+
+На первом занятии Сильвен вместе с вами изучит Главу 1 курса, объясняя ее шаг за шагом:
+
+<div class="flex justify-center">
```

Comment:
```suggestion
```

Reply:
The videos don't render in the course both in RU version, and in EN version. I think it's because of the <div>